samples/template/Kconfig.defconfig

```diff
@@ -60,8 +60,10 @@ config SHELL_STACK_SIZE
 	default 8192 if TEMPLATE_APP_CLI
 
 config HEAP_MEM_POOL_SIZE
-	default 8192 if SIDEWALK_ON_DEV_CERT
-	default 1024
+	default 4096
+
+config MBEDTLS_HEAP_SIZE
+	default 4096
```

Comment:
I do not think we should have this in the template's Kconfig, 
IMO the better place would be the sidewalk/Kconfig.dependencies under the CONFIG_SIDEWALK_CRYPTO, because this change is needed always when we need to use CONFIG_SIDEWALK_CRYPTO, and not just in template

Reply:
Agree, but when defined in Kconfig, the default value is covered at 512 from other kconfig